example-scripts/bash-scripts/full-harvest-examples/1.6-examples/example-csv/vivo.model.xml

```diff
@@ -100,7 +100,7 @@
 <Model>
     <Param name="type">sdb</Param>
     <Param name="dbLayout">layout2</Param>
-    <Param name="dbClass">com.mysql.jdbc.Driver</Param>
+    <Param name="dbClass">com.mysql.cj.jdbc.Driver</Param>
```

Comment:
how this line is linked with h2 upgrade?

Reply:
Not sure if it is directly related to H2, but the harvester log said:
"Loading class `com.mysql.jdbc.Driver'. This is deprecated. The new driver class is `com.mysql.cj.jdbc.Driver'."
So I changed the line accordingly.

Reply:
ok, let's keep that in this PR